packages/j-uppercase/src/index.ts

```diff
@@ -1 +1,9 @@
-export default "J";
+/**
+ * @link https://en.wikipedia.org/wiki/J
+ * I bet you didn't read through all of these
```

Comment:
```suggestion
 * I bet you did not read through all of these
```
As this is in the documentation for the uppercase character J, I suspect many individuals reading this will not be familiar with advanced concepts such as [Punctuation](https://en.wikipedia.org/wiki/Punctuation) and [Apostrophes](https://en.wikipedia.org/wiki/Apostrophe). To ensure the documentation remains readable to beginners of the alphabet, I would encourage you to remove this use of an [Apostrophe](https://en.wikipedia.org/wiki/Apostrophe).